scripts/deploy_depeg.py

```diff
@@ -969,20 +973,27 @@ def inspect_applications(d):
             policy = None
             kind = 'application'
 
-        print('{} {} {} {} {} {} {:.1f} {:.1f} {} {:.1f}'.format(
+        print('{} {} {} {} {} {} {} {:.1f} {:.1f} {} {:.1f}'.format(
             idx,
-            customer[:6],
+            _shortenAddress(customer),
             productId,
             processId,
             kind,
             state,
+            _shortenAddress(wallet),
             premium/mul_usd2,
             suminsured/mul_usd1,
             duration/(24*3600),
             maxpremium/mul_usd2
         ))
 
 
+def _shortenAddress(address) -> str:
+    return '{}..{}'.format(
+        address[:5],
```

Comment:
last 4 are use in most cases

Reply:
already using the first 5 and last 4 (same as metamask)